readthedocs.yml

```diff
@@ -2,4 +2,9 @@ version: 2
 
 mkdocs:
   configuration: packages/docs/sdk/mkdocs.yml
-  fail_on_warning: true
\ No newline at end of file
+  fail_on_warning: true
```

Comment:
what if we didnt fail on warning?

Reply:
that's an option. seems better to make packages explicit and fail on warnings though, so we can address them as they come up.